app/src/ui/history/commit-summary.tsx

```diff
@@ -384,13 +396,15 @@ export class CommitSummary extends React.Component<
       return null
     }
 
+    const sortedTags = [...tags].sort(caseInsensitiveCompare)
+
     return (
-      <li className="commit-summary-meta-item">
+      <li className="commit-summary-meta-item" title={sortedTags.join('\n')}>
```

Comment:
i think a comma here will make this more useful for screenreaders
```suggestion
      <li className="commit-summary-meta-item" title={sortedTags.join(',\n')}>
```

Reply:
I've used newlines to separate the tags in the tooltip, so commas won't play nice with that... I wonder if I can make screenreaders ignore the title (since it contains exactly the same as the text inside the `<li>`).

Reply:
looks like we're fine to leave as-is after reading https://developer.paciellogroup.com/blog/2013/01/using-the-html-title-attribute-updated/

> Takeaway: If you want to hide content from mobile and tablet users as well as assistive tech users and keyboard only users, use the title attribute.